phalcon/assets/resource.zep

```diff
@@ -61,16 +61,22 @@ class $Resource implements ResourceInterface
 
 	protected _targetUri { get };
 
+	protected _version { set, get };
+
+	protected _autoVersion = false { set };
+
 	/**
 	 * Phalcon\Assets\Resource constructor
 	 */
-	public function __construct(string type, string path, boolean local = true, boolean filter = true, array attributes = [])
+	public function __construct(string type, string path, boolean local = true, boolean filter = true, attributes = null, var version = null, var autoVersion = null)
```

Comment:
@Jurigag Do we really need to change attributes type? The Phalcon v4 will have a lot of BC incompatible changes. Could you please try to sort out how to avoid yet another